portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,8 +36,20 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println(convertToJsonUsingGson(comments));
+    
+    final Query allComments = new Query("Comment");
+    DatastoreService dataStore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = dataStore.prepare(allComments);
+
+    ArrayList<String> commentList = new ArrayList<>();
+    for (Entity entity : results.asIterable()) {
+        final String comment = (String) entity.getProperty("comment-field");
```

Comment:
Create a global constant for any string you would use in getProperty(). This helps with reuse and preventing typo errors.

`public static final COMMENT_FIELD = "comment-field";`
`entity.getProperty(COMMENT_FIELD);`